test_runner/src/main/kotlin/ftl/cli/hypershard/android/HyperAndroidCommand.kt

```diff
@@ -0,0 +1,103 @@
+package ftl.cli.hypershard.android
+
+import com.dropbox.mobile.hypershard.ClassAnnotationValue
+import com.dropbox.mobile.hypershard.RealHyperShard
+import ftl.util.FlankFatalError
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import picocli.CommandLine.Command
+import picocli.CommandLine.Option
+import java.io.File
+import java.lang.Runnable
+import java.nio.file.Paths
+
+private const val RESULT_FILE = "hypershard-android-results.txt"
+
+@Command(
+    name = "android",
+    sortOptions = true,
+    headerHeading = "",
+    synopsisHeading = "%n",
+    descriptionHeading = "%n@|bold,underline Description:|@%n%n",
+    parameterListHeading = "%n@|bold,underline Parameters:|@%n",
+    optionListHeading = "%n@|bold,underline Options:|@%n",
+    header = ["Run hypershard-android against test classes"],
+    description = ["Hypershard is a fast and simple test collector that uses the Kotlin and Java ASTs. " +
+            "Flank will save to file full qualified test names found in dir(s)" +
+            " that can be used later as input for --test-targets option."
+    ],
+    usageHelpAutoWidth = true
+)
+class HyperAndroidCommand : Runnable {
```

Comment:
Let's include an example in the readme.

Reply:
Definitely, this PR is still in progress.